src/main/com/daniel/example1/RandomGeneratorApplication.java

```diff
@@ -0,0 +1,26 @@
+package main.com.daniel.example1;
```

Comment:
No chtěl jsem ať upravíš ten svůj kód, tady to vypadá jak to moje, ale nevadí. Už tak nech, ať se posuneme. I tak ti to okomentuji.